lab-justin/lib/storage.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
```

Comment:
The bulk of this code is repeated checks, made inside each method that the same things are present. Can you think of a way to DRY it out?